W-17167450 RPA Manager release notes 2024-11/1. (#433)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[november-13-2024]]
== November 13, 2024

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | Interval scheduling now shows correct times for all time zones of the login user. | W-16739470

Reviewer: ```suggestion
| Interval scheduling now shows the correct times for all time zones of the logged-in user. | W-16739470
```

> | Interval scheduling now shows the correct times for all time zones of the logged-in user. | W-16739470

Reviewer: ```suggestion
| Interval scheduling no longer shows incorrect times for the time zones of the logged-in user. | W-16739470
```

> | The edit action of activity parameters within a run configuration is selectable again. | W-16821955

Reviewer: ```suggestion
| Activity parameters within a run configuration are no longer blocked from editing. | W-16821955
```

> | The list of run configurations in the production phase is now sortable by the _Type_ column. | W-16887965

Reviewer: ```suggestion
| The list of run configurations in the production phase no longer sorts incorrectly when clicking the _Type_ column. | W-16887965
```
